Use dump-as-markup.js in editing/inserting/typing-003.html

rniwa opened the pull request:
#### 548b61da69dc6a7ccaa25325b37f9635a711bd2d
<pre>
Use dump-as-markup.js in editing/inserting/typing-003.html
<a href="https://bugs.webkit.org/show_bug.cgi?id=252610">https://bugs.webkit.org/show_bug.cgi?id=252610</a>

Reviewed by Wenson Hsieh.

Use dump-as-markup.js in this test for clarity.

* LayoutTests/editing/inserting/typing-003-expected.txt:
* LayoutTests/editing/inserting/typing-003.html:
* LayoutTests/platform/gtk/editing/inserting/typing-003-expected.png: Removed.
* LayoutTests/platform/gtk/editing/inserting/typing-003-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/editing/inserting/typing-003-expected.png: Removed.
* LayoutTests/platform/mac/editing/inserting/typing-003-expected.png: Removed.
* LayoutTests/platform/wincairo/editing/inserting/typing-003-expected.txt: Removed.

Canonical link: <a href="https://commits.webkit.org/260592@main">https://commits.webkit.org/260592@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/028b84cad2ea0df3b186b1c782e771da3c4fdd1c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/108733 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/17836 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/41582 "Built successfully") | [❌ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/19 "Updated wpe dependencies (failure)") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118051 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/19285 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9111 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/100971 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/114501 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14471 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/97695 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/42444 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96445 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29333 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/84290 "Found 2 new API test failures: /TestWebKit:WebKit.OnDeviceChangeCrash, /TestWebKit:WebKit.PrivateBrowsingPushStateNoHistoryCallback (failure)") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/10632 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/30684 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11394 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/7611 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/16781 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50274 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/12980 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/3986 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

whsieh commented on LayoutTests/editing/inserting/typing-003-expected.txt:
```diff
@@ -745,17 +745,7 @@ EDITING DELEGATE: webViewDidChange:WebViewDidChangeNotification
-    RenderBody {BODY} at (8,8) size 784x584
-      RenderBlock {DIV} at (0,0) size 784x140 [border: (2px solid #FF0000)]
-        RenderText {#text} at (14,14) size 738x112
-          text run at (14,14) width 702: "xxxx x xx xxxx xxxxxx xxxxx xxxxxx xxx xx xxxx xxx xxx xx x xxxxx "
-          text run at (14,42) width 738: "xxxxxxxxxxxx xxxxxxxx xxx xxxx xxxxx xx xxx xxxx xxxx xxxxx xxxxxx "
-          text run at (14,70) width 720: "xxx xx xxxx xxx xxx xx xxxxxx xxxxx xxxxxx xxx xx xxxx xxx xxx xx x "
-          text run at (14,98) width 354: "xxxxx xxxxxxxxxxxx xxxxxxxx xxx"
-          text run at (368,98) width 6: " "
```
So it looks like one aspect of this test that might be lost in the transition to `dump-as-markup.js` are the words on each individual text run, which seem (*almost*) consistent across the different baselines — only in the GTK result, the offsets of text runs 2-5 seem to be different. Do you think it's worth preserving this part of the test? (potentially, in such a way that it would only check what text is present in each of the 5 text runs).

rniwa replied:
I don't think we need to preserve that. As far as I can tell, the core of this test is the typed text matching the expectation, not so much that specific line contains specific characters.

rniwa committed:
Use dump-as-markup.js in editing/inserting/typing-003.html
https://bugs.webkit.org/show_bug.cgi?id=252610

Reviewed by Wenson Hsieh.

Use dump-as-markup.js in this test for clarity.

* LayoutTests/editing/inserting/typing-003-expected.txt:
* LayoutTests/editing/inserting/typing-003.html:
* LayoutTests/platform/gtk/editing/inserting/typing-003-expected.png: Removed.
* LayoutTests/platform/gtk/editing/inserting/typing-003-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/editing/inserting/typing-003-expected.png: Removed.
* LayoutTests/platform/mac/editing/inserting/typing-003-expected.png: Removed.
* LayoutTests/platform/wincairo/editing/inserting/typing-003-expected.txt: Removed.

Canonical link: https://commits.webkit.org/260592@main
